### DISCUSSION TITLE
Getting several nextcloud office errors

### DISCUSSION THREAD

**Discussion by rspierenburg** [nextcloud/all-in-one#4845] [Questions] [answered]
I am constantly getting this time out error pointing to Rich Documents.
_Failed to connect to the remote server: cURL error 28: Operation timed out after 5002 milliseconds with 0 bytes received (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://www.xxxxxxxxx.ca/hosting/capabilities_

I've tried to follow the debug steps here https://github.com/nextcloud/all-in-one/discussions/1358 but when I visit the Rich Documents admin page I don't see anything about "Allow list ofr WOPI requests"

I am running behind a Caddy reverse proxy but according to the documentation Caddy V2 enable websockets out of the box.

[Hosting Discovery.txt](https://github.com/user-attachments/files/15900647/Hosting.Discovery.txt)
[Collabora Logs.txt](https://github.com/user-attachments/files/15900683/Collabora.Logs.txt)

I apologize, but I find it all a bit confusing.  Am I correct in assuming that Rich Documents is tied to Nextcloud Office which is tied to Collabora?  I'm just wondering which apps should be enabled in Nextcloud.

Robert

**Comment by szaimen** [comment, marked as answer]
> I've tried to follow the debug steps here #1358 but when I visit the Rich Documents admin page I don't see anything about "Allow list ofr WOPI requests"

Hi, are you sure? It should be `https://yourdomain.com/settings/admin/richdocuments` which you need to visit. For me it looks like this:
![image](https://github.com/nextcloud/all-in-one/assets/42591237/67ca4b92-c9af-4834-8ffc-7e420f386f28)

**Comment by Snuupy** [comment]
I'm running into this as well. You can't change the `Allow list for WOPI requests` if you also have the error `Could not establish connection to the Collabora Online server.` so it seems like a catch 22.

my issue is that I need to set a static IP for my instance and then in /etc/hosts put it in

setting 127.0.0.1 nextcloud.domain.tld didn't work, it needs some type of external IP (whether from LAN or WAN for some reason)

**Reply by micker** [reply]
hello did you have a solution ?

**Reply by Snuupy** [reply]
@micker set the whitelist to 0.0.0.0

**Reply by micker** [reply]
By édition host file ?
